### PR TITLE
updates to SME quick start guide

### PR DESCRIPTION
**Pull request by srweintraub**

### CODE REVIEW

**Comment by srweintraub** [comment]
@cklunch PR to update the quick start guide for SME/PLFA

**Comment by srweintraub** [comment]
I added rows for the scaled table in the Table Joining section, hopefully this is correct? I did not add a row for joining sme_microbialBiomass to sme_scaledMicrobialBiomass, but maybe I should?

**Comment by srweintraub** [comment]
I added some proposed updates to the SLS 10086 QSG as well, including a new figure. Let me know what you think.

File: DP1.10086.001/Table.joining.md [hunk 1]
```diff
@@ -9,15 +9,15 @@ ntr_internalLab|ntr_internalLabBlanks|kclReferenceID
...
-ntr_externalLab|sls_soilChemistry|Direct join not recommended: nitrogen incubations are carried out on a different set of samples from other chemical analyses
+ntr_externalLab|sls_soilChemistry|sampleID
 ntr_externalLab|sls_soilCoreCollection|sampleID
 ntr_externalLab|sls_soilMoisture|sampleID
 ntr_externalLab|sls_soilpH|sampleID
-ntr_internalLab|sls_bgcSubsampling|Direct join not recommended: nitrogen incubations are carried out on a different set of samples from other chemical analyses
+ntr_internalLab|sls_bgcSubsampling|sampleID
 ntr_internalLab|sls_metagenomicsPooling|Direct join not recommended: nitrogen incubations are carried out on a different set of samples from genomic sequencing
-ntr_internalLab|sls_soilChemistry|Direct join not recommended: nitrogen incubations are carried out on a different set of samples from other chemical analyses
+ntr_internalLab|sls_soilChemistry|sampeID
```
Comment (cklunch) [inline DP1.10086.001/Table.joining.md:20]: Missing "l" in sampleID

**Review by cklunch — COMMENTED** [review]
@srweintraub Looks great to me! Just caught one typo, see comment.

File: DP1.10086.001/Table.joining.md (hunk 1 above)
Comment (srweintraub) [reply DP1.10086.001/Table.joining.md]: sorry and thanks for catching! I pushed a fix.

**Pull request merged**